styles/mixins/_lists.scss

```diff
@@ -3,6 +3,8 @@ Lists mixins - Co-op Front-end Toolkit
 ===============================================*/
 
 @mixin list {
+  @include font-size('body');
```

Comment:
One last thing: shouldn't be necessary to include the `font-size('body')` mixin as we're including those in the `body` mixin, which gets applied to the list HTML element in /styles/_typography` -- which is the same behaviour for all other text elements (headings, paragraphs, etc).